code/modules/hydroponics/grown.dm

```diff
@@ -72,15 +86,93 @@
 		return
 	return
 
+
+// Various gene procs
+/obj/item/weapon/reagent_containers/food/snacks/grown/attack_self(mob/user)
+	if(seed.get_gene(/datum/plant_gene/trait/squash))
+		squash(user)
+	..()

```

Comment:
Same as with glow case, make it generic, what if someone will want to add new mutation that is activated here